nncf/quantization/algo.py

```diff
@@ -821,17 +825,20 @@ def __init__(self, target_model: NNCFNetwork,
             self.metric_store = {}
             quantizer_setup_type = self.quantization_config.get('quantizer_setup_type')
             # These metrics are collected here and are updated when the method .statistics() is called
-            self.non_stable_metric_collectors = [NetworkQuantizationShareMetric(target_model, self.weight_quantizers,\
-                 self.non_weight_quantizers, quantizer_setup_type), MemoryСostMetric(target_model, self.weight_quantizers, self.non_weight_quantizers)]
+            self.non_stable_metric_collectors = [NetworkQuantizationShareMetric(target_model, self.weight_quantizers, \
```

Comment:
Why the `non_stable_metric_collectors ` variable is called so? Are there any problems with these collectors?

Reply:
@asenina